src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.body {
+  margin: 0;
+  --best-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav,
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
https://github.com/mate-academy/layout_moyo-header/pull/2912#discussion_r1153406210

Reply:
I do not understand what do you mean. I did it again but I don't know if it's right